.firebaserc

```diff
@@ -1,5 +1,5 @@
 {
   "projects": {
-    "default": "whereshouldweeattoday"
+    "default": "what-should-we-eat"
   }
 }
```

Comment:
Whoops. Didn't realize you created one (with the actual correct name, lol). You can keep it as yours. I just created one cause I wanted to work on some better Auth / login boilerplate this morning